_posts/2022-10-14-athena.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "Trino delivers for Athena"
+author: Manfred Moser, Martin Traverso
+image: /assets/trino-light.png
+excerpt_separator: <!--more-->
+---
+
+Today it is time to reach out to another large group of Trino users. The release
+of the [Amazon Athena](https://aws.amazon.com/athena) engine 3 upgrades Athena
+to a recent version of Trino. This massive update brings a tons of improvements
+the Trino project to the large userbase of the popular cloud-based query
+service.
+
+<!--more-->
+
+## Shared history
+
+Amazon Athena and Trino share a long history. From the beginning of Athena the
+query engine under the hood was Trino, then still called Presto. Athena created
+a low-maintenance, powerful access mode to your data in S3 and beyond. It
+combined the performance and features of Trino, with the convenience of a cloud
+service, and therefore enabled new users and use cases. You could take advantage
+of Trino, without needing a team of experts being have to deployment and operate
+a Trino cluster. In fact, we wrote about this even the first edition of **Trino:
+The Definitive Guide**, and there is also a section in the [new second
+edition]({% post_url 2022-10-03-the-definitive-guide-2 %}), that you can get for [free from Starburst](https://www.starburst.io/info/oreilly-trino-guide/).
+
+## Time flies
+
+Since the initial release however, time has not stood still. In fact the Trino
+project has accelerated in [innovation, features, and releases tremendously]({%
+post_url 2022-08-04-decade-innovation %}). Amazon Athena users now get access to
+many of these great features. As [AWS mentions in the
+announcement](https://aws.amazon.com/about-aws/whats-new/2022/10/amazon-athena-announces-upgraded-query-engine/),
+"over 50 new SQL functions, 30 new features, and more than 90 query performance
+improvements" are now available due the upgrade to a new version of Trino. These
+include great features like [Row pattern recognition with MATCH_RECOGNIZE]({%
+post_url 2021-05-19-row_pattern_matching %}), [new window features]({% post_url
+2021-03-10-introducing-new-window-features %}), and support for ``UPDATE`` or
+``TRUNCATE`` statements.
+
+Performance improvements in our core engine as well as all the Trino connector,
+show up in every release note. The [improvements observed by the Athena team in
+their
+benchmarks](https://aws.amazon.com/blogs/big-data/upgrade-to-athena-engine-version-3-to-increase-query-performance-and-access-more-analytics-features/)
+are show the resulting gains nicely. This is great evidence that our approach of
+constantly working on small improvements wherever we find potential works well.
+This is necessary since we already at a very high performance level, like an
+elite athlete, every small improvement matters.
+
```

Comment:
Can you add the latest number of commits graph comparing the commits between Trino v Presto? @martint has the script that generates it.

Reply:
Not sure I want to even go there and shine more light on Presto..

Reply:
That's fair, the only reason I think to bring it up **One Last Time™** is that our target audience for this post is largely people who use Athena that are less aware of Presto or Trino. 

We want them to learn that 
  a) Trino is under the hood and there are more options than just Athena.
  b) Of the two technology names associated with Presto/Trino, they should use Trino.
  
So I'm kind of for either solution. We have to name Presto in this article anyways so saying the name twice and showing how we beat it isn't a big difference to me anyways from giving the Presto name internet points.

Reply:
Let me figure out how to add that in